hack/remove-sbr-finalizers.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+SBR=$(oc get sbrs)
```

Comment:
do we really need to use `oc` here or `kubectl` is enough? Asking because at line 6 we use `kubectl`. Let's consolidate.

We could save us one call here. How about the following:

```shell
#!/bin/bash -xe

SBRS=$(oc get sbrs -n $TEST_NAMESPACE -o jsonpath="{.items[*].metadata.name}")
for i in $SBRS; do
     kubectl patch sbr/$i -p '{"metadata":{"finalizers":[]}}' --type=merge;
done
```

mind `-xe` option on bash - `-e` exists the script if any any command returns non-zero exit code and `-x` displays on the console what commands are executed.